app/scrapers/quora.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import re
+from bs4 import BeautifulSoup
+
+class Quora:
+    """Scrapper class for quora"""
+    def __init__(self):
+        pass
+
+    def get_page(self,query):
+        """ Fetch the quora search results page
+        Returns : Results Page
+        """
+        query_string = '-'.join(query.strip(' ').split(' '))
+        response = requests.get('https://www.quora.com/%s' %(query_string))
```

Comment:
I was just looking at quora and I found https://www.quora.com/search?q=<query> as the search link?

Reply:
That was for questions containing terms, when you go to a specific question with multiple answers, the query is what I have used. I assumed that people using the service for quora would be using it to display answers for the question/query and not question(s) for the query.